src/components/Navbar.tsx

```diff
@@ -8,15 +15,29 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+          <NavLink
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'has-background-grey-lighter': isActive },
+            )}
+            to="/"
+            onClick={() => (setSelectedSlug
+              ? setSelectedSlug('')
+              : '')}
```

Comment:
`setSelectedSlug(setSelectedSlug(''))` ??? What did you want to do here? Refactor it, please